recipes/cmec_driver/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "cmec_driver" %}
+{% set version = "1.0.0a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cmecmetrics/cmec-driver/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5620acc7787b155c4f7c3e5922681e958528f23dd41170a28c7dbb90771fe184
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```

I think your package is pure python.  If so, it's best to do:
```suggestion
  noarch: python
```
even if you don't intend to support Windows.  It saves a lot of redundant builds.  It's okay if the Windows build fails.

---

You also need to add your entry point here:
```
  entry_points:
    - cmec-driver = cmec_driver:main
```
I'm a little surprised the tests worked without this.

---

We should improve the linter to catch that one but the tests will pass on Linux. Only Windows require us to explicitly define the entry points.